src/components/StructuredContent/StructuredContent.stories.tsx

```diff
@@ -0,0 +1,512 @@
+          "Pellentesque massa placerat duis ultricies lacus sed turpis tincidunt. " +
+          "<a href='#'>Porttitor eget dolor</a> morbi non arcu risus quis varius. " +
+          "Faucibus in ornare quam viverra orci sagittis.</p><p>Nisl vel pretium " +
+          "lectus quam id leo in. Etiam dignissim diam quis enim lobortis " +
+          "scelerisque fermentum. Diam maecenas sed enim ut sem viverra. <b>Diam " +
+          "quam nulla porttitor massa id neque. Sed faucibus turpis in eu mi. " +
+          "</b></p><p>Ornare lectus sit amet est placerat in. Quis blandit turpis " +
+          "cursus in. Aliquam ut porttitor leo a diam sollicitudin tempor id eu. " +
+          "Pellentesque eu tincidunt tortor aliquam nulla facilisi cras fermentum. " +
+          "Porttitor leo a diam sollicitudin tempor id eu nisl nunc. Feugiat" +
+          "nisl pretium fusce id velit ut tortor. Porttitor leo a diam sollicitudin " +
+          "tempor id eu nisl nunc.</p><ul><li>one</li><li>two</li>" +
+          "<li>three</li><li>four</li></ul><p><i>Mauris nunc congue nisi vitae suscipit" +
+          "tellus mauris a diam. Purus gravida quis blandit turpis cursus in hac." +
+          "Morbi tempus iaculis urna id volutpat. Lectus nulla at volutpat diam ut" +
+          "venenatis. Donec ac odio tempor orci dapibus ultrices in iaculis. Dui vivamus" +
+          "arcu felis bibendum ut tristique. Cras semper auctor neque vitae tempus" +
+          "quam pellentesque. Placerat orci nulla pellentesque dignissim enim sit" +
+          "amet. Feugiat pretium nibh ipsum consequat. Placerat orci nulla pellentesque" +
+          "dignissim. Suspendisse faucibus interdum posuere lorem. Nullam non nisi" +
+          "est sit. Turpis egestas integer eget aliquet nibh praesent. Tortor at risus" +
+          "viverra adipiscing at. Eu augue ut lectus arcu bibendum at varius vel" +
+          "pharetra.</i></p>"
+        }
+      />
+      <StructuredContent
+        calloutText="This is the callout text"
+        headingText="Heading text"
+        bodyContent={
+          "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do " +
+          "eiusmod tempor incididunt ut labore et dolore magna aliqua. " +
+          "Pellentesque massa placerat duis ultricies lacus sed turpis tincidunt. " +
+          "<a href='#'>Porttitor eget dolor</a> morbi non arcu risus quis varius. " +
+          "Faucibus in ornare quam viverra orci sagittis.</p><p>Nisl vel pretium " +
+          "lectus quam id leo in. Etiam dignissim diam quis enim lobortis " +
+          "scelerisque fermentum. Diam maecenas sed enim ut sem viverra. <b>Diam " +
+          "quam nulla porttitor massa id neque. Sed faucibus turpis in eu mi.</b></p>"
+        }
+      />
+    </>
+  ),
+};
+export const ExampleWithThreeComponents: Story = {
+  render: () => (
+    <>
+      <StructuredContent
+        bodyContent="This is the body text"
```

Comment:
The original example didn't include `bodyContent`, so it was able to rely on the spacing from the callout to add space between the first and second `<StructuredContent>` components.  With `bodyContent` in there, the spacing is now gone.  Howeverr, the spacing issue can be corected by wrapping the `bodyContent` text in a `<p>` element.

Reply:
The nice thing about this migration is that it is catching errors that did not show up in MDX. Yes, the original example did not include `bodyContent` but it is actually a required prop. So in this `.tsx` file, this came up as an error in the IDE. I can all the `<p>` element to fix this example, but I don't think it's safe to assume that devs will be able to wrap content in a `<p>` element if spacing is off. So perhaps this is a bug that _we_ should fix?

Reply:
Yes, I will create a ticket and assign it to myself.

Reply:
Actually, no, we should not add default spacing after the component.  We don't do that for other components and we expect consuming apps to use soemthing like `VStack` to add the necessary spacing.